resources/assets/app.css

```diff
@@ -17,4 +17,26 @@
 @tailwind components;
 @tailwind utilities;
 
-[x-cloak] { display: none !important; }
+.table-of-contents {
+    @apply pb-3;
+}
+
+.table-of-contents > li {
+    @apply my-[0.35rem];
+}
+
+.table-of-contents ul {
+    @apply pl-2;
+}
+
+.table-of-contents a {
+    @apply block -ml-8 pl-8 opacity-80 hover:opacity-100 hover:bg-gray-200/20 transition-all duration-300 relative;
+}
+
+.table-of-contents a::before {
+    @apply content-['#'] text-[75%] opacity-50 mr-1 transition-opacity duration-300;
+}
+
+.table-of-contents a:hover::before {
+    @apply opacity-100;
+}
```

Comment:
I think later I want to refactor the table of contents generation to be Blade-based. The generator can get the data needed, and a customizable Blade partial is used to add the styles.

Reply:
Now tracked in https://github.com/hydephp/develop/pull/2006

Reply:
Fixed in https://github.com/hydephp/develop/pull/2045